### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create new file_Screen Reader: Narrator/NVDA is not reading the label for "Kind" and "Source Folder" fields

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4282]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ScreenReader; #NVDA; #Narrator; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator, NVDA

**MAS Violated:** MAS1.3.1

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Module Dependencies" tab and press enter
8.Tab till "Finish" button and press enter
9.In Project Explorer, expand the created Scala project right-click src, New-->Other and press enter
10.Tab till Scala Wizards, expand and select 'Scala Object' and press next
11.Check if the screen reader is reading the labels for "Kind" and "Source Folder" fields

**Actual Result:**
Narrator and NVDA are not reading the label for "Kind" and "Source Folder" fields

**Expected Result:**
Narrator and NVDA should read the label for "Kind" and "Source Folder" fields

**User impact:**
Screen Reader users will not understand the purpose of the control if the label is not read 

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true)

![Bug_4282_Java Tooling_Create new file_Labels_NVDA](https://user-images.githubusercontent.com/41108985/78987528-e4d2b800-7b4b-11ea-98a8-b14c81668789.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit.

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Important. Screen Reader users will not understand the purpose of the control if the label is not read.
+ Quality of experience: Manageable. A brief training on scala knowledge could let user figure the purpose of the control from the given value. And it is not block the create process
+ Alternate solutions:  Provide scala training information.
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562988

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749644
